sklearn/cluster/mean_shift_.py

```diff
@@ -155,13 +159,18 @@ def mean_shift(X, bandwidth=None, seeds=None, bin_seeding=False,
                 break  # Depending on seeding strategy this condition may occur
             my_old_mean = my_mean  # save the old mean
             my_mean = np.mean(points_within, axis=0)
-            # If converged or at max_iter, addS the cluster
+            # If converged or at max_iter, adds the cluster
             if (extmath.norm(my_mean - my_old_mean) < stop_thresh or
                     completed_iterations == max_iter):
                 center_intensity_dict[tuple(my_mean)] = len(points_within)
                 break
             completed_iterations += 1
 
+    if not center_intensity_dict:
+        # nothing near seeds
+        raise ValueError("No point was within bandwidth of any seed."
```

Comment:
Can you please recall the actual value of the `bandwidth` parameter in this error message?